vagrant-manager: Add version 2.3.2

absalomedia opened the pull request:
Add GUI based Vagrant Manager to Scoop

Ash258 commented on bucket/vagrant-manager.json:
```diff
@@ -0,0 +1,21 @@
+    "autoupdate": {
+        "url": "https://github.com/absalomedia/vagrant-manager/releases/download/$version/vagrant-manager.exe"
+    }
+}
```
```suggestion
}

```

Ash258 commented on bucket/vagrant-manager.json:
```diff
@@ -0,0 +1,21 @@
+    "checkver": {
+        "github": "https://github.com/absalomedia/vagrant-manager",
+        "regex": "/tag/([-\\w.]+)"
+    },
```
```suggestion
    "checkver": "github",
```

Ash258 commented on bucket/vagrant-manager.json:
```diff
@@ -0,0 +1,21 @@
+ {
+    "version": "2.3.2",
+    "description": "Vagrant Manager",
```
```suggestion
    "description": "Status bar menu for managing all Vagrant machines from one central location",
```

Ash258 commented:
Why did you select exactly this fork?

Ash258 commented on bucket/vagrant-manager.json:
```diff
@@ -0,0 +1,21 @@
+    "autoupdate": {
+        "url": "https://github.com/absalomedia/vagrant-manager/releases/download/$version/vagrant-manager.exe"
+    }
+}
```
```suggestion
    "suggest": {
        "Vagrant": "vagrant"
    },
    "url": "https://github.com/absalomedia/vagrant-manager/releases/download/2.3.2/vagrant-manager.exe#/dl.7z",
    "hash": "c6a99b2e5b18f384ac5b25abc539c319efe9b3cd472c9b73a01130826b1f6240",
    "architecture": {
        "64bit": {
            "installer": {
                "script": "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\""
            }
        },
        "32bit": {
            "installer": {
                "script": "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-32.7z\" \"$dir\""
            }
        }
    },
    "post_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse",
    "shortcuts": [
        [
            "Vagrant Manager.exe",
            "Vagrant Manager"
        ]
    ],
    "checkver": "github",
    "autoupdate": {
        "url": "https://github.com/absalomedia/vagrant-manager/releases/download/$version/vagrant-manager.exe#/dl.7z"
    }
```

absalomedia replied:
I downloaded the vagrant-manager-2.3.2-x64.nsis.7z package to verify it. Extracted out fine. Ran fine. Windows 10 Pro running v1809. 

The Vagrant Manager.exe file inside the NSIS 7z file did what it was supposed to do. End up in the Windows taskbar. It's not an installer, it's the final app.

Ash258 replied:
Yes I know it is not installer... And yes it is executed in taskbar, but right click do not work. It do nothing

- Sandbox 1903
- Server 2019
- Server 2012R2
- Windows 20H1
- Windows 1909

absalomedia replied:
I'll check on 1909 once the feature update goes through on my machine tonight.

Ash258 replied:
Apply this changes, since current manifest is not working.

absalomedia replied:
Applied changed to JSON based on your feedback

absalomedia commented:
> Why did you select exactly this fork?

I'm the maintainer for the fork. The project I forked off was not maintained so in that time I've:

- rewritten underlying foundations & logic
- implemented cross platform build processes (Windows, OSX, Linux), including automated version releases. 7z files aren't meant to be captured as part of Travis yet Github is somehow doing that. EXE, DMG, ZIP, DEB files are the expected artifacts with the Travis build sequence.
- translated the app into multiple languages
- extended out the base functionality to include a much larger set of Vagrant related functions

absalomedia commented:
@Ash258 just for future reference, what is the hash based on? The 64-bit 7z or the 32bit?

r15ch13 commented:
Why not use `vagrant-manager-x64.7z` and `vagrant-manager-ia32.7z` instead of the whole installer (which is double the size)?

Ash258 commented:
https://github.com/lukesampson/scoop-extras/pull/3567#discussion_r375411907 I cannot make it work in my environemnt

r15ch13 commented:
okay 😄

Ash258 commented:
I will look into it in future, but right now it is not needed.
